perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -39,6 +43,76 @@
 log = logging.getLogger()
 
 
+def _upload_to_gsheet(worksheet, data, spreadsheet_id) -> (int):
```

Comment:
I think we have this function on listing_benchmark also. Can we move that in to util?

Reply:
Due to difference in function behavior, making this function private to rename benchmark.